tests/test_electrolyzer.py

```diff
@@ -151,6 +155,19 @@ def test_electrolyzer_inequality(self):
                 results["Electrolyzer_fc66.ElectricityIn.Power"] * a[i] + b[i] + 1.0e-3,
             )
 
+        # Check electrolyzer input power
+        np.testing.assert_allclose(
+            results["Electrolyzer_fc66.ElectricityIn.Power"],
+            [1.00000000e08, 1.00000000e08, 1.00000000e08],
+            atol=1e-4,
+        )
```

Comment:
Ideally I would, you could check the bound on the Power of the electrolyzer, if that is equal to 1e8 and then check if the results is close to the bound. In case test case would be changed in numbers, it easier to change/check/traceback.
One tip, if it is a constant value you want to check upon, no need to create a list, but just use 1.0e8.

Reply:
The bound is 5e8